python/semantic_kernel/connectors/ai/function_calling_utils.py

```diff
@@ -120,6 +120,4 @@ def merge_streaming_function_results(
 
     # If we want to be able to support adding the streaming message chunks together, then the author role needs to be
     # `Assistant```, as the `Tool` role will cause the add method to break.
```

Comment:
The comment is not updated.

Reply:
Removed comment as it no longer applies.